schemas/WebpackOptions.json

```diff
@@ -828,10 +828,6 @@
         "syncWebAssembly": {
           "description": "Support WebAssembly as synchronous EcmaScript Module (outdated).",
           "type": "boolean"
-        },
-        "topLevelAwait": {
```

Comment:
Removing an option is a breaking change. This will break builds for developers using this option. Although this is under experiments, I don't feel we should remove this. Maybe just normalize this to `true` by default if not defined in user's webpack config.

@alexander-akait @TheLarkInn What do you think?

Reply:
Gotcha. Makes sense

Reply:
Yes, we can't remove `topLevelAwait`, we should just enable `topLevelAwait` by default, using `topLevelAwait: false` should disable it, also please add `TODO webpack v6, remove it`

Reply:
This is fine to merge, we'll release as SemVer minor for tomorrow.